civihr_default_theme/scss/overrides/_civicrm.scss

```diff
@@ -1,7 +0,0 @@
-// With the inclusion of custom-civicrm.css in onboarding-form,
-// a top padding of 60px is added making space for menubar.
-// But we dont need menubar for obnoarding form.
-// So, the padding is set back to 0px.
-body:not(.toolbar) {
-  padding-top: 0 !important;
```

Comment:
Have you run backstop after that? That looks like a major change for me.

Reply:
I tried, but backstop tests does not work in civihr, so, it didn't get tested. However, i was decided with discussion with Alessandro, it was never needed when we don't use 'custom-civicrm.css" in onboarding-form page for datepicker styles.